readthedocs/search/documents.py

```diff
@@ -81,9 +81,9 @@ def faceted_search(
         }
 
         filters = {}
-        if projects_list:
+        if isinstance(projects_list, list) and projects_list:
             filters['project'] = projects_list
-        if versions_list:
+        if isinstance(versions_list, (list, str)) and versions_list:
```

Comment:
This one was tricky because of the `_list` suffix. `versions_list` can be a list or just a `latest` string, for example. I was expecting it to behave as `projects_list` that it's a list of one element in that case.